### ISSUE TITLE
mysql errors out after pasting the new configuration file

### ISSUE THREAD

**Issue by kmalinoski** [matecat/MateCat#2576]
Not sure where to go from here, this is the message i receive after trying to start the service. If I revert back to the old mysql config file, it starts no problem:

Job for mysql.service failed because the control process exited with error code.
See "systemctl status mysql.service" and "journalctl -xeu mysql.service" for details.

**Comment by kmalinoski** [comment]
@Ostico - would you be able to assist in this installation error? I am following the steps in this article: https://site.matecat.com/installation-guide/

**Comment by Jemikalem** [comment]
I am facing exactly the same error and neither the systemctl status neither the journalctl gives relevants logs errors in order to dig further : 

**Logs from the journalctl** 
```
-- The job identifier is 6271 and the job result is failed.
Mar 02 17:08:45 ubuntu-focal systemd[1]: mysql.service: Start request repeated too quickly.
Mar 02 17:08:45 ubuntu-focal systemd[1]: mysql.service: Failed with result 'exit-code'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- The unit mysql.service has entered the 'failed' state with result 'exit-code'.
Mar 02 17:08:45 ubuntu-focal systemd[1]: Failed to start MySQL Community Server.
-- Subject: A start job for unit mysql.service has failed
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- A start job for unit mysql.service has finished with a failure.
```

**Logs from the systemctl status**
```
root@ubuntu-focal:/etc/mysql/mysql.conf.d# systemctl status mysql.service
● mysql.service - MySQL Community Server
     Loaded: loaded (/lib/systemd/system/mysql.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Thu 2023-03-02 17:19:36 UTC; 3s ago
    Process: 34961 ExecStartPre=/usr/share/mysql/mysql-systemd-start pre (code=exited, status=1/FAILURE)

Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Failed with result 'exit-code'.
Mar 02 17:19:36 ubuntu-focal systemd[1]: Failed to start MySQL Community Server.
Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Scheduled restart job, restart counter is at 5.
Mar 02 17:19:36 ubuntu-focal systemd[1]: Stopped MySQL Community Server.
Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Start request repeated too quickly.
Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Failed with result 'exit-code'.
Mar 02 17:19:36 ubuntu-focal systemd[1]: Failed to start MySQL Community Server.
```

Any help should be wonderful. Thanks in advance.

**Comment by golden-h** [comment]
> I am facing exactly the same error and neither the systemctl status neither the journalctl gives relevants logs errors in order to dig further :
> 
> **Logs from the journalctl**
> 
> ```
> -- The job identifier is 6271 and the job result is failed.
> Mar 02 17:08:45 ubuntu-focal systemd[1]: mysql.service: Start request repeated too quickly.
> Mar 02 17:08:45 ubuntu-focal systemd[1]: mysql.service: Failed with result 'exit-code'.
> -- Subject: Unit failed
> -- Defined-By: systemd
> -- Support: http://www.ubuntu.com/support
> --
> -- The unit mysql.service has entered the 'failed' state with result 'exit-code'.
> Mar 02 17:08:45 ubuntu-focal systemd[1]: Failed to start MySQL Community Server.
> -- Subject: A start job for unit mysql.service has failed
> -- Defined-By: systemd
> -- Support: http://www.ubuntu.com/support
> --
> -- A start job for unit mysql.service has finished with a failure.
> ```
> 
> **Logs from the systemctl status**
> 
> ```
> root@ubuntu-focal:/etc/mysql/mysql.conf.d# systemctl status mysql.service
> ● mysql.service - MySQL Community Server
>      Loaded: loaded (/lib/systemd/system/mysql.service; enabled; vendor preset: enabled)
>      Active: failed (Result: exit-code) since Thu 2023-03-02 17:19:36 UTC; 3s ago
>     Process: 34961 ExecStartPre=/usr/share/mysql/mysql-systemd-start pre (code=exited, status=1/FAILURE)
> 
> Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Failed with result 'exit-code'.
> Mar 02 17:19:36 ubuntu-focal systemd[1]: Failed to start MySQL Community Server.
> Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Scheduled restart job, restart counter is at 5.
> Mar 02 17:19:36 ubuntu-focal systemd[1]: Stopped MySQL Community Server.
> Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Start request repeated too quickly.
> Mar 02 17:19:36 ubuntu-focal systemd[1]: mysql.service: Failed with result 'exit-code'.
> Mar 02 17:19:36 ubuntu-focal systemd[1]: Failed to start MySQL Community Server.
> ```
> 
> And infortunately no more relevant information in the **errors.log**
> 
> Any help should be wonderful. Thanks in advance !

hi, i had the same problem.
I solved by reinstalling on Ubuntu 18.04.
Another alternative is to find a way to install mysql v5.7.36 on the OS of your choice (I'm guessing you chose ubuntu 22 or later).
Hope to help you

**Comment by Ostico** [comment]
Stale
